lib/fastlane/plugin/wpmreleasetoolkit/helper/strings_file_validation_helper.rb

```diff
@@ -0,0 +1,129 @@
+module Fastlane
+  module Helper
+    class StringsFileValidationHelper
```

Comment:
Ah! I kept it generic because it seemed clear to me "strings file" referred to `.strings`. I hadn't considered the `strings.xml` naming convention in Android 🤦‍♂️  Thanks.